src/index.js

```diff
@@ -233,7 +233,7 @@ export default (options = {}) => {
           cssOptions.to = codeFileName
         }
 
-        const result = await require('cssnano').process(code, cssOptions)
+        const result = await require('cssnano')(postcssLoaderOptions.minimize).process(code, cssOptions)
```

Comment:
With cssnano 5, you need to call `cssnano()`, pass cssnano specific options to it. This returns a normal PostCSS processor, which you then call to process the CSS.